lib/webapp.js

```diff
@@ -365,10 +365,10 @@ const staticHandlers = {
     const respondToGetPlugins = function(req, res, webapp, type, plugins) {
       const acceptLanguage = 
             translationUtils.getAcceptLanguageFromCookies(req.cookies) || req.headers['accept-language'] || '';
-      const pluginDefs = plugins.map(p => p.exportTranslatedDef(acceptLanguage));
+      const pluginDefs = plugins.filter(p => p.identifier!='org.zowe.zss.swagger').map(p => p.exportTranslatedDef(acceptLanguage));
```

Comment:
We use 'org.zowe.zss.swagger' in a few different places, having one constant that gets imported from a JS file may be better

Reply:
made lot of adjustments, so don't have to explicitly use the name of plugin everywhere.